Framework/DataHandling/src/LoadILLSANS.cpp

```diff
@@ -759,7 +766,7 @@ void LoadILLSANS::setFinalProperties(const std::string &filename) {
   NXstatus nxStat = NXopen(filename.c_str(), NXACC_READ, &nxHandle);
 
   if (nxStat != NX_ERROR) {
-    m_loader.addNexusFieldsToWsRun(nxHandle, runDetails);
+    m_loadHelper.addNexusFieldsToWsRun(nxHandle, runDetails);
```

Comment:
I assume this is run at the very end, are we sure that this doesn't override the start time already set in ISO format?